src/main/java/ch/njol/skript/conditions/CondIsDancing.java

```diff
@@ -0,0 +1,40 @@
+package ch.njol.skript.conditions;
+
+import ch.njol.skript.conditions.base.PropertyCondition;
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Examples;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.Since;
+import org.bukkit.entity.Allay;
+import org.bukkit.entity.LivingEntity;
+import org.bukkit.entity.Parrot;
+
+@Name("Is Dancing")
+@Description("Checks to see if an entity is dancing. (e.g allays and parrots)")
+@Examples({
+	"if last spawned allay is dancing:",
+		"\tbroadcast \"Dance Party!\""
+})
+@Since("INSERT VERSION")
+public class CondIsDancing extends PropertyCondition<LivingEntity> {
+
+	static {
+		PropertyCondition.register(CondIsDancing.class, PropertyType.BE, "dancing", "livingentities");
```

Comment:
```suggestion
		PropertyCondition.register(CondIsDancing.class, "dancing", "livingentities");
```
the type is `BE` by default

Reply:
also you can just do `register(...)` instead of `PropertyCondition.register(...)`

Reply:
Hmmm, I'm pretty sure I tried that before at one point and it didn't show. I could be wrong.